Volha_Ivanova/Module_4_Framework/Design_Patterns/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/fileHandlerFactory/FileHandler.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.cdp.volha_ivanova.task1.fileHandlerFactory;
```

Comment:
Please change package to fileHandler. The package can contains FileHandlers and Factory.
On the future: if you do even education task please try to do it as production code. I think you choose the name of the package to show me where factory is. But honestly it's not the best idea. Your code should have good structure to can find appropriate class.